src/constants/request-states.js

```diff
@@ -21,23 +21,23 @@ export const KYC_REQUEST_STATES = Object.freeze(
   Object.entries({
     approved: {
       state: REQUEST_STATES.approved,
-      label: 'Approved',
+      label: 'request-states.approved',
```

Comment:
label  - > translationId
fix the place where used

Reply:
done